spec/spec_helper.rb

```diff
@@ -117,9 +117,14 @@ def restart_phantomjs
   # Enable caching in any specs tagged with `caching: true`. Usage is exactly the same as the
   # well-known `js: true` tag used to enable javascript in feature specs.
   config.around(:each, :caching) do |example|
-    caching = ActionController::Base.perform_caching
+    caching = Rails.configuration.action_controller.perform_caching
+
+    Rails.configuration.action_controller.perform_caching = example.metadata[:caching]
     ActionController::Base.perform_caching = example.metadata[:caching]
```

Comment:
have you verified if you still need  ActionController::Base.perform_caching  here?

Reply:
It was needed for the other PRs, yes.

Reply:
I meant if the other PRs work only with Rails.configuration.action_controller.perform_caching